packages/components/pie-form-label/src/index.ts

```diff
@@ -6,6 +6,11 @@ import { property } from 'lit/decorators.js';
 import styles from './form-label.scss?inline';
 import { FormLabelProps } from './defs';
 
+type PIEInputElement = Partial<HTMLInputElement> & {
```

Comment:
this is simply a PoC interface that could represent what kinds of methods and properties our own input components expose to consumers